sdks/python/apache_beam/io/avroio.py

```diff
@@ -601,15 +602,42 @@ def display_data(self):
     return res
 
 
-class _FastAvroSink(_AvroSink):
-  """A sink for avro files that uses the `fastavro` library"""
+class _FastAvroSink(filebasedsink.FileBasedSink):
```

Comment:
Most of this code is duplicated from _AvroSink.
It might be better to create a _BaseAvroSink which they both inherit from.